boards/shared/CMakeLists.txt

```diff
@@ -42,7 +42,10 @@ set(SHARED_IO_INCLUDE_DIRS ${SHARED_IO_INCLUDE_DIRS} PARENT_SCOPE)
 file(GLOB_RECURSE ARM_BINARY_APP_SRCS "Src/App/*.*")
 set(ARM_BINARY_APP_INCLUDE_DIRS "Inc/App")
 
-file(GLOB GOOGLETEST_TEST_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/Test/Src/*.cpp")
+file(GLOB GOOGLETEST_TEST_SRCS
```

Comment:
Aren't the GTest source files always going to be in `.cpp` because it's a C++ framework? Not sure why this change is necessary.

Reply:
In this case I had an odd issue where I had to make the `TestWorld.c` file a `.c` otherwise I was getting undefined references, because we include `TestWorld.h` in an `extern C`. I think this is good because it's consistent with how we create and include `World`'s elsewhere, but open to changing it if you feel otherwise.

Reply:
Oh I see the dilemma now. I feel like since `TestWorld.c` is an `App` file it would make more sense to append it to `ARM_BINARY_APP_SRCS`, i.e.:

```
list(APPEND ARM_BINARY_APP_SRC "Src/App/App_TestWorld.c)
```
And leave this line as-is. What do you think?

Reply:
@garethellis0 Think you missed this comment

Reply:
But that's a test file, it should never be built into the arm binary, no?

Reply:
Anyhow, I toyed with this a bit more and got it to work as a `.cpp`, so removed the `*.c` match.

Reply:
Oh yeah I see what you're saying, i'm fine with the either way then: either replacing `.c` with `.cpp` or adding `"${CMAKE_CURRENT_SOURCE_DIR}/Test/Src/*.c"` back.